### PR TITLE
merge v1 into main with manual updates

### PR DESCRIPTION
**Pull request by ivansenic**
**What this PR does**:

Merges `v1` into main.. Unfortunately, I had to do a lot of manual checks and adaptations in this merge. I guess the problem is in the move to the `coordinator/` folder that was done on `main` and some files were not picked up correctly (a lot did).

We made a mistake of not doing the mode to the coordinator in v1 first. I'll try to fix this, but I am not sure how successful I will be.

**MUST BE MERGED WITH FAST FORWARD**.

* [ ] fully disable GCB on merge

### CODE REVIEW

File: coordinator/testing/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Stargate Testing
+
+This module provides integration tests for the Stragate Coordinator.
```
Comment (jeffreyscarpenter) [inline coordinator/testing/README.md:3]: spelling error "Stragate"

**Review by jeffreyscarpenter — APPROVED** [review]
one minor spelling error

**Comment by jeffreyscarpenter** [comment]
> Merges `v1` into main.. Unfortunately, I had to do a lot of manual checks and adaptations in this merge. I guess the problem is in the move to the `coordinator/` folder that was done on `main` and some files were not picked up correctly (a lot did).
> 
> We made a mistake of not doing the mode to the coordinator in v1 first. I'll try to fix this, but I am not sure how successful I will be.

What if we reverted the move to the `coordinator/` folder on main and started over with v1? That might end up being the least painful solution.

**Comment by ivansenic** [comment]
Not needed anymore, closing.

**Pull request closed without merging**